tests/components/unifi/test_init.py

```diff
@@ -140,33 +90,8 @@
 
 async def test_unload_entry(hass):
     """Test being able to unload an entry."""
-    entry = MockConfigEntry(
-        domain=unifi.DOMAIN,
-        data={
-            "controller": {
-                "host": "0.0.0.0",
-                "username": "user",
-                "password": "pass",
-                "port": 80,
-                "site": "default",
-                "verify_ssl": True,
-            },
-            "poe_control": True,
-        },
-    )
-    entry.add_to_hass(hass)
+    controller = await setup_unifi_integration(hass)
+    assert hass.data[unifi.DOMAIN]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4925854)